package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "se-2",
+  "name": "se-2-noir",
   "version": "0.0.1",
```

Comment:
maybe we should change this to `1.0.0`? :)

Reply:
Good point.... mmm I think this should be 0.1.0 actually

Reply:
OK, sure.

Reply:
Done